### PR TITLE
Fix : Edit section 1 and section 2 of the readme inside @oyenuga17/command-line folder

### PR DESCRIPTION
**Pull request by oyenuga17**
**This pull request makes the following changes:**
* Fixes issue #108 

**Checklist:**
- [x] I have created an issue for this solution I am submitting
- [x] I created a new branch from the correct branch and made my changes in the new branch created
- [x] My pull request is sent to the correct base branch in the original git-github-training repo
- [x] I have linked the issue I worked on to this pull request submitted by me
- [x] File(s) or folder(s) now contain changes as specified in the issue i worked on
- [x] I certify that I ran my checklist.

Ping @Ifycode/git-github-training

### CODE REVIEW

**Comment by Ifycode** [comment]
Hi @oyenuga17. Using command line, do the following:
- Checkout to your own branch
- Merge develop into your own branch with: `git merge develop`
- That is when you will be able to merge the conflict. Add/commit your changes, and the push your commit.

**Comment by oyenuga17** [comment]
i have made my changes please review @Ifycode

File: @oyenuga17/command-line/README.md [hunk 1]
```diff
@@ -2,7 +2,10 @@
...
 ## Section 1: How to submit your changes to this repo with Github UI
-You can watch this video to learn how to add content to a repo that is not yours, through the Github UI: [YouTube tutorial](https://www.youtube.com/watch?v=2X1FjFqIwIY).
+Just watch this video to learn how to add content to a repo that is not yours, through the Github UI: [YouTube tutorial](https://www.youtube.com/watch?v=2X1FjFqIwIY).
 
 ## Section 2: Github markdown resources
-The resources below show you how to use markdown: This resource tells you [how to format content using github markdown](https://gist.github.com/cuonggt/9b7d08a597b167299f0d) and you can also click this to [download github markdown cheat sheet in pdf](https://ifycode.github.io/git-github-training/markdown-cheatsheet.pdf).
+The resources below show you how to use markdown: 
+- [how to format content using github markdown](https://gist.github.com/cuonggt/9b7d08a597b167299f0d)
+- [download github markdown cheat sheet in pdf](https://ifycode.github.io/git-github-training/markdown-cheatsheet.pdf).
+
```
Comment (Ifycode) [inline @oyenuga17/command-line/README.md:11]: Nice one @oyenuga17 😉👍

**Review by Ifycode — APPROVED** [review]

File: @oyenuga17/command-line/README.md (hunk 1 above)
Comment (Ifycode) [reply @oyenuga17/command-line/README.md:11]: Merge lesson task successfully completed 💯

**Pull request merged**